test/cli-options.test.ts

```diff
@@ -1,14 +1,12 @@
-'use strict'
-
-const chai = require('chai')
-const cliOptions = require('../src/cli-options').default
+import chai from 'chai'
+import cliOptionsSorted from '../src/cli-options'
```

Comment:
`cliOptions` is a better name than `cliOptionSorted` here since the sorting is not relevant and we can hide that detail from the test component.

Reply:
I agree, but I can't find the right syntax to import a default import with a different name. If you can, I'll be happy to change this!

Reply:
```ts
import cliOptions from '../src/cli-options'
```

Reply:
Fixed in fcf2ab1

Reply:
Thanks, my IDE kept showing errors when I tried that, but it looks like it works :)